### ISSUE TITLE
Incomplete page: "Using the aria-activedescendant attribute":

### ISSUE THREAD

**Issue by Uziel302** [mdn/content#3989]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-activedescendant_attribute

#### I see the word "code"

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/aria_techniques/using_the_aria-activedescendant_attribute`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-activedescendant_attribute
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/aria_techniques/using_the_aria-activedescendant_attribute/index.html
* Last commit: https://github.com/mdn/content/commit/2d777fc9792270755b8b105e381b482a274e2707
* Document last modified: 2020-12-23T14:44:27.000Z

</details>

**Comment by chrisdavidmills** [comment]
This page is in draft form, and needs to be completed. In particular, it is missing an example, and other important background information. Whoever attempts this task would need to do a bunch of research to find the correct information to add.

**Comment by estelle** [comment]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-activedescendant
